ocaml/idl/datamodel.ml

```diff
@@ -6576,6 +6576,18 @@ let vdi_disable_cbt = call
     ~doc:"Disable changed block tracking for the VDI. This call is only allowed on VDIs that support enabling CBT. It is an idempotent operation - disabling CBT for a VDI for which CBT is not enabled results in a no-op, and no error will be thrown."
     ~allowed_roles:_R_VM_ADMIN
     ()
+(**This command is for internal use by SM to set the cbt_enabled field when it needs to disable cbt it for its own reasons*)
+(**This command should be removed once SMAPIv3 is implemented*)
```

Comment:
It's better to have this as one comment (also, fixed the typo "disable cbt it"):
```
(** This command is for internal use by SM to set the cbt_enabled field when it
    needs to disable CBT for its own reasons.
    This command should be removed once SMAPIv3 is implemented. *)
```
Sorry for being so pedantic, the only reason I remarked this is that the comments with the double star are special ocamldoc comments, and I'm not sure how these are merged together, so it's better to have both lines in one comment.

Reply:
Its no worries. I will change it now